tests/cache/models/samples/upsertBulk.js

```diff
@@ -358,20 +357,26 @@ describe('api::redisEnabled::POST::bulkUpsert ' + path, () => {
         },
       ])
       .then(() => {
-        api.get('/v1/samples?name=' +
-          `${tu.namePrefix}Subject|${tu.namePrefix}Aspect1`)
-        .set('Authorization', token)
-        .end((err, res) => {
-          if (err) {
-            done(err);
-          }
+        /*
+         * the bulk api is asynchronous. The delay is used to give sometime for
+         * the upsert operation to complete
```

Comment:
What if we check the GET /.../status endpoint in a while loop until status is complete, and *then* try to GET the sample?